--- cmake/external_dependencies.cmake
+++ cmake/external_dependencies.cmake
@@ -14,13 +14,8 @@
 
 # Encryption control
 if(NOT NO_ENCRYPTION)
-    if(PLATFORM_WINDOWS)
-        set(ENABLE_BCRYPT_ENCRYPTION ON)
-    elseif(PLATFORM_LINUX OR PLATFORM_ANDROID)
-        set(ENABLE_OPENSSL_ENCRYPTION ON)
-    elseif(PLATFORM_APPLE)
-        set(ENABLE_COMMONCRYPTO_ENCRYPTION ON)
-    endif()
+    # conda defaults: force use of OpenSSL for all platforms
+    set(ENABLE_OPENSSL_ENCRYPTION ON)
 else()
     set(ENABLE_INJECTED_ENCRYPTION ON)
 endif()
